packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -72,10 +72,11 @@ interface MuxUploaderElement extends HTMLElement {
 
 class MuxUploaderElement extends globalThis.HTMLElement implements MuxUploaderElement {
   static get observedAttributes() {
-    return ['no-drop', 'no-progress', 'no-status', 'no-retry', 'max-file-size'];
+    return ['pausable', 'no-drop', 'no-progress', 'no-status', 'no-retry', 'max-file-size'];
   }
 
   protected _endpoint: Endpoint;
+  protected _upload?: UpChunk;
```

Comment:
NOTE: need to keep this in an instance level prop now for pause/resume functionality.